wordpress-mobile/gutenberg-mobile
# Automate gutenberg translations followup

This PR aims to improve our i18n flow. There are 3 main improvements we need to take care of:
- [x] Fix plurals on android. GlotPress supports `<string-array>` tags for plurals instead of `<plurals>`
- [x] Find a way to keep the same ref/name for each string so not all strings are updated when we refresh the translation files.
- [x] Exclude strings from tests files in gutenberg
 
### Testing Instructions
- Make sure the gutenberg submodule is updated
- Run `yarn genstrings`
- Make sure there are no changes to string files in bundles (`git diff`)

Update release notes:
